CONTRIBUTING.md

```diff
@@ -160,43 +148,6 @@ The following list shows some guiding rules that you should keep in mind when yo
 
 > Note: you might notice that some of the topics are not currently following all the guidelines specified here and on the [style guide](./styleguide/template.md) as well. We're working towards achieving consistency throughout the site. Check the list of [open issues](https://github.com/dotnet/docs/issues?q=is%3Aissue+is%3Aopen+label%3Aguidelines-adherence) we're currently tracking for that specific goal.
 
-## Building the docs
```

Comment:
What if my preferred workflow involves testing changes locally before pushing? Are we discouraging that practice?

Reply:
At this time, yes. There are a lot of issues with the local build: You need to download the dependent repos, and have them in the right location. Xref doesn't work reliably for most community members. #627 has some of the details.